Editorial fix for #2479 (#2877)
quicwg/base-drafts: draft-ietf-quic-invariants.md

The next four bytes include a 32-bit Version field (see {{version}}).

> The byte contains the length in bytes of the Destination Connection ID (see

Reviewer: ```suggestion
The next byte contains the length in bytes of the Destination Connection ID (see
```

{{connection-id}}) field that follows it.  This length is encoded as an 8-bit
unsigned integer.  The Destination Connection ID field follows the DCID Len
field and is between 0 and 255 bytes in length.

> The byte following the Source Connection ID contains the length in bytes

Reviewer: ```suggestion
The next byte contains the length in bytes
```

The next four bytes include a 32-bit Version field (see {{version}}).

The byte contains the length in bytes of the Destination Connection ID (see
{{connection-id}}) field that follows it.  This length is encoded as an 8-bit

> unsigned integer.  The Destination Connection ID field follows the DCI Len

Reviewer 2: ```suggestion
unsigned integer.  The Destination Connection ID field follows the DCID Len
```

> and is between 0 and 255 bytes in length.

Reviewer 2: ```suggestion
field and is between 0 and 255 bytes in length.
```

The byte following the Source Connection ID contains the length in bytes
of the Source Connection ID field that follows it.  This length is encoded as
a 8-bit unsigned integer.  The Source Connection ID field follows the SCID Len

> and is between 0 and 255 bytes in length.

Reviewer 2: ```suggestion
field and is between 0 and 255 bytes in length.
```

> {{negotiating-connection-ids}} describes the use of the source and destination

Reviewer 2: This is what is causing CI to fail.  This anchor is in a different document.
